stuff/lint.sh

```diff
@@ -32,7 +32,8 @@ if [[ -d /proc ]] && grep -q pids:/docker /proc/1/cgroup; then
 	exit 1
 fi
 DOCKER_PATH="$(which docker)"
-if [[ -z "${DOCKER_PATH}" ]]; then
+if [[ !"${DOCKER_PATH}" || -z "${DOCKER_PATH}" ]]; then
+	# Validate `which` error code and empty path.
```

Comment:
```suggestion
if [[ $? != 0 || -z "${DOCKER_PATH}" ]]; then
```

esta es la manera correcta multi-plataforma de verificar que el comando anterior falló.

Reply:
Gracias por la sugerencia.